docs/api/app.md

```diff
@@ -554,11 +554,16 @@ Returns `Promise<void>` - fulfilled when Electron is initialized.
 May be used as a convenient alternative to checking `app.isReady()`
 and subscribing to the `ready` event if the app is not ready yet.
 
-### `app.focus()`
+### `app.focus([force])`
+
+* `force` String (optional) _macOS_ - Make the receiver the active app even if another app is
```

Comment:
Might seem overkill but can we make this. `app.focus({ force: true })` better readability and expandability

Reply:
yeah, we can avoid the boolean trap here :)